src/_lib/sass/styles.scss

```diff
@@ -1,32 +1,32 @@
 //== Import Bootstrap mixins
 //
 // Utilities
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/hide-text";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/opacity";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/image";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/responsive-visibility";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/reset-text";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/text-emphasis";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/text-overflow";
-@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/vendor-prefixes";
+@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/hide-text";
```

Comment:
Not sure what caused these paths to break, but simply saying `node_modules/...` no longer works.